<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="playerChoice">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Rock, Paper, or Scissor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scissorsButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>112</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Scissors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rockButton">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>50</y>
      <width>112</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paperButton">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>50</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Paper</string>
    </property>
   </widget>
   <widget class="QLabel" name="oppChoice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oppPic">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>100</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerChoicePic">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameResult">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>300</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
